build.gradle

```diff
@@ -191,6 +191,18 @@ dependencyManagement {
     dependencySet(group: 'com.google.guava', version: '29.0-jre') {
       entry 'guava'
     }
+    // solves CVE-2020-5407
```

Comment:
I think we suppressed this in other apps since we are no using - can you check and keep consistent please?

Reply:
I have added suppression but still it has spring-security-config/web/core that has to be suppressed

Reply:
I modified cpe to include other spring-security-*